addons/mail/static/src/components/activity/tests/activity_tests.js

```diff
@@ -788,7 +788,7 @@ QUnit.test('activity upload document is available', async function (assert) {
     });
     this.data['mail.activity'].records.push({
         activity_category: 'upload_file',
-        activity_type_id: 1,
+        activity_type_id: 28,
```

Comment:
As said before, this becomes a magic number without explicit activity type data defined in this test.

Reply:
However I agree for the part on not using a magic ID. What is done in this case on the server is using the "xml id" to find the proper record without hard-coding id, which we are lacking here.